source/cre-mappings.yaml

```diff
@@ -0,0 +1,725 @@
+---
+meta: {component: mappings, edition: ecommerce, language: ALL, version: '1.20'}
+suits:
+- cards:
+  - capec: [54, 541]
+    cre: [308-515]
+    description: Brian can gather information about the underlying configurations,
+      schemas, logic, code, software, services and infrastructure due to the content
+      of error messages, or poor configuration, or the presence of default installation
+      files or old, test, backup or copies of resources, or exposure of source code
+    owasp_appsensor: [HT1-3]
```

Comment:
These descriptions are in the cards files so that translation is easier, they should not be included in the mappings files as they'll not get used to generate the cards, unless they are needed to create the CRE mapping?

Reply:
i was using them as a visual queue to double check the mapping is correct, now that we have the mappings I'll remove them, thanks!